src/SourceBuild/content/eng/finish-source-only.proj

```diff
@@ -114,12 +114,22 @@
           Outputs="$(BaseIntermediateOutputPath)ReportPoisonUsage.complete" >
     <ItemGroup>
       <!-- Exclude the Private.SourceBuilt.Artifacts archive from poison usage scan. -->
-      <PoisonFileToCheck Include="$(ArtifactsAssetsDir)*$(ArchiveExtension)" />
-      <PoisonFileToCheck Remove="$(ArtifactsAssetsDir)$(SourceBuiltArtifactsTarballName)*" />
+      <AssetToCheck Include="$(ArtifactsAssetsDir)*$(ArchiveExtension)" />
+      <AssetToCheck Remove="$(ArtifactsAssetsDir)$(SourceBuiltArtifactsTarballName)*" />
```

Comment:
nit: Instead of the additional `Remove` operation, I would add an `Exclude` to L117.

Reply:
Thanks. I'd like to minimize the changes in this backport, especially as this was already the model before my PR.

Reply:
Sure. Feel free to consider this for the PR targeting main.